src/Components/Calculator.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, useState } from "react";
+import CSS from "csstype";
+import Display from "./Display";
+import Numbers from "./Numbers";
+import Operators from "./Operators";
+import isOperator from "../helpers/isOperator";
+
+const Calculator: FC = () => {
+  const [display, setDisplay] = useState("");
+  const lastChar = display[display.length - 1];
+
+  const handleClick = (el: string): void => {
+    // design choice: if operator is repeating, record the last operator clicked
+    if (display.length > 0 && isOperator(lastChar) && isOperator(el)) {
+      setDisplay((prevDisplay: string): string => prevDisplay.slice(0, -1));
+    }
+
+    setDisplay((prevDisplay: string): string => prevDisplay + el);
+  };
+
+  const handleSubmit = (): void => {
+    /* design choice: do nothing if user has inserted invalid equation, 
+    assume they haven't finished */
+    if (display.length > 0 && !isOperator(lastChar)) {
+      setDisplay((prevDisplay: string): string => eval(prevDisplay).toString());
```

Comment:
`eval` is the simplest way to do the calculation logic but it's also best avoided. There's probably a library that you can use instead.